### PR TITLE
Cdms fields

### PR DESCRIPTION
**Pull request by marcofucci**
New CDMS declarative fields instead of lambda to convert values from/to cdms

### CODE REVIEW

**Comment by marcofucci** [comment]
@danielquinn code review?

File: crm-poc/apps/cdms_api/utils.py [hunk 1]
```diff
@@ -1,40 +1,29 @@
+import re
 import time
-from unittest import mock
+import datetime
 
-from django.utils import timezone
 
-from .base import CDMSApi
+datetime_re = re.compile('/Date\(([-+]?\d+)\)/')
```
Comment (danielquinn) [inline crm-poc/apps/cdms_api/utils.py]: This should probably be `DATETIME_RE` (or even better, `DATETIME_REGEX`) since it's a constant.

Comment (marcofucci) [inline crm-poc/apps/cdms_api/utils.py]: agree, Django (https://github.com/django/django/blob/master/django/utils/dateparse.py#L23) uses lowercase but not sure why

**Comment by danielquinn** [comment]
Normally, any project that contains a file called `utils.py` sets off alarm bells in my head, but I see you've namespaced it to be specifically for CDMS translations, so I guess it's ok.

Everything looks good to me.

**Pull request merged**